From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: 5jd <85830207+5jd@users.noreply.github.com>
Date: Wed, 19 Oct 2022 13:17:53 +0200
Subject: [PATCH] Fix Petal issues


diff --git a/src/main/java/net/minecraft/server/MinecraftServer.java b/src/main/java/net/minecraft/server/MinecraftServer.java
index abc5a4a67270db2b801d02fe870e73b448fac9a7..76bcbdafb462a37eac1d43088dca873f3cd004e7 100644
--- a/src/main/java/net/minecraft/server/MinecraftServer.java
+++ b/src/main/java/net/minecraft/server/MinecraftServer.java
@@ -1683,14 +1683,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
 
     @DontObfuscate
     public String getServerModName() {
-        // petal start
-        final String purpurModName = org.purpurmc.purpur.PurpurConfig.serverModName; // Purpur - Purpur > // Paper - Paper > // Spigot - Spigot > // CraftBukkit - cb > vanilla!
-
-        return switch (purpurModName) {
-            case "Purpur" -> "Filter+";
-            default -> purpurModName;
-        };
-        // petal end
+        return org.purpurmc.purpur.PurpurConfig.serverModName; // Bee - Bring back Purpur serverModName // Purpur - Purpur > // Paper - Paper > // Spigot - Spigot > // CraftBukkit - cb > vanilla!
     }
 
     public SystemReport fillSystemReport(SystemReport details) {
